### PR TITLE
feat: add basic smoke tests

### PR DESCRIPTION
**Pull request by hugo-gomes**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -6,13 +6,20 @@ jobs:
...
     strategy:
       matrix:
-        node-version: [14, 15, 16, 17, 18, 19]
+        node-version: [18, 19, 20, 21]
     runs-on: ubuntu-latest
     container: node:${{ matrix.node-version }}
     steps:
       - uses: actions/checkout@v1
+      - run: apt-get update && apt-get install -y wget gnupg
+      - run: >-
```
[the remark this answers is not in the document]
Comment (hugo-gomes) [reply .github/workflows/main.yml:15]: when the unit tests ran, `initEngines` did not have a chance to execute Puppeteer's setup
having Chrome pre-installed provides a unit test environment ready for test execution

[the remark this answers is not in the document]
Comment (joamag) [reply .github/workflows/main.yml:15]: But Pupeteer is not able to install Chrome? that's weird...

[the remark this answers is not in the document]
Comment (joamag) [reply .github/workflows/main.yml:15]: it should be

[the remark this answers is not in the document]
Comment (joamag) [reply .github/workflows/main.yml:15]: https://pptr.dev/#:~:text=When%20you%20install%20Puppeteer%2C%20it,guaranteed%20to%20work%20with%20Puppeteer.

image

**Pull request merged**